tests/spec/README.md

```diff
@@ -0,0 +1,112 @@
+# Spec tests
+
+These tests specify the expected behaviour of ehrQL, and by extension, the QM.
```

Comment:
suggestion:

```suggestion
These tests specify the expected behaviour of ehrQL, and by extension, the query model.
```

We're not using this abbreviation elsewhere in this document, so I'd remove it to make the reader's life easier.

(I realise it was here already. But if we're improving the whole document in this PR, we might as well do this too.)